api/index.py

```
@@ -1,6 +1,7 @@
 import os
 import datetime
 from api import *
```

Ruff (0.8.0)

`from api import *` used; unable to detect undefined names

(F403)